api/placer.py

```diff
@@ -504,5 +504,21 @@ def finalize(self):
             metadata_info = metadata_infos.get(info['name'], {})
             metadata_info.update(info)
             self.metadata['files'].append(metadata_info)
-        self.metadata['_id'] = str(bson.objectid.ObjectId())
         return self.metadata
+
+class AnalysisJobPlacer(AnalysisPlacer):
+    def check(self):
+        super(AnalysisJobPlacer, self).check()
+        self.metadata['outputs'] = self.metadata['acquisition'].pop('files', [])
```

Comment:
I've talked with @kofalt recently about making the metadata output more generic, but yeah definitely a hack "for now" 😎